main.py

```diff
@@ -26,14 +26,31 @@
 
 
 def main(is_simulation: bool, real_servo: bool) -> None:
+    """
+    The main function that will be run when the program is started. It will create the objects that will be used in the
+    airbrakes context and run the main loop. The main loop will run until the user presses Ctrl+C.
+
+    This function handles is what handles if the program is running in simulation mode or not.
+
+    :param is_simulation: Whether to run the program in simulation mode or not
+    :param real_servo: Whether to use the real servo or a mock servo
+    """
     # Create the objects that will be used in the airbrakes context
+    sim_time_start = time.time()
```

Comment:
this will be done in the other PR, no need to duplicate

Reply:
wait so what do you want me to do

Reply:
delete that line in this pr, it already exists, and this is a redefinition